rs/registry/canister/tests/add_node.rs

```diff
@@ -399,7 +399,7 @@ fn init_mutation_with_rewards_table() -> RegistryAtomicMutateRequest {
     RegistryAtomicMutateRequest {
         mutations: vec![RegistryMutation {
             mutation_type: registry_mutation::Type::Insert as i32,
-            key: NODE_REWARDS_TABLE_KEY.as_bytes().to_vec(),
+            key: NODE_REWARDS_TABLE_KEY.into(),
```

Comment:
`.into()` is not preferred, because it is as explicit. In particular, it does not name the destination type (`to_vec` does).